buildpipeline/portable-linux.groovy

```diff
@@ -61,11 +61,13 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
                                      'Ubuntu.1404.Amd64.Open',
                                      'Ubuntu.1604.Amd64.Open',
                                      'Ubuntu.1610.Amd64.Open',
-                                     // 'Ubuntu.1704.Amd64.Open',
                                      'suse.422.amd64.Open',
                                      'fedora.25.amd64.Open',]
             if (params.OuterLoop) {
-                targetHelixQueues += ['Debian.90.Amd64.Open',]
+                targetHelixQueues += ['Debian.90.Amd64.Open',
```

Comment:
What is the logic in picking these as the outerloop ones? In my view of the world it would seem like the best option would be to include just the latest versions of the important distro's in the interloop tests and everything else in outerloop.